docs/compare_collections.md

```diff
@@ -56,15 +56,15 @@ Now that you know the basics, and once you have an understanding of what the com
 
 ## Limitation of the comparison function: distinguishing out-of-order from mismatched arrays
 
-One limitation of the comparison function is that it does comparisons at the level of arrays, not at the level of individual elements. What would the comparison function return for two sequence collections that are exact duplicates except for two component sequences that have swapped names?
+One limitation of the comparison function is that it does comparisons at the level of arrays, not at the level of individual elements. What would the comparison function return for two sequence collections that have the same content, but in different orders, AND where in addition two of the sequences have swapped names?
 
 Because the sequence array would contain the same sequences, the comparison function will count them all as matching.
 Similarly, the names arrays contain the same names and so all will be counted as a match.
-However, the same_order will *not* be true, so it will yield false.
+However, the same_order will *not* be true; it will yield false for all attributes.
```

Comment:
This is again incorrect. Here's an example:

https://refget.databio.org/comparison/fLf5M0BOIPIqcfbE6R8oYwxsy-PnoV32/E6zGtGuc8wKYmCMw5gaLW3ppyXsoO6p4

it does not yield false for all attributes. The lengths and sequences are in the same order.

Reply:
so it will yield false only for the `names` attribute.

Reply:
I agree